### PR TITLE
DBZ-7616 Document `database.query.timeout.ms` property

### PR DESCRIPTION
**Pull request by mfvitale**
related to: https://github.com/debezium/debezium/pull/5483

### CODE REVIEW

File: documentation/modules/ROOT/pages/connectors/db2.adoc [hunk 1]
```diff
@@ -3044,6 +3044,10 @@ User intervention is required to restart the connector.
 `> 0`:: The connector restarts automatically until it reaches the specified maximum number of retries.
 After the next failure, the connector stops, and user intervention is required to restart it.
 
+|[[db2-property-database-query-timeout-ms]]<<db2-property-database-query-timeout-ms, `database.query.timeout.ms`>>
+|`600000`
+|Time to wait for a query to execute, given in milliseconds. Defaults to 600 seconds (600,000 ms); zero means there is no limit.
```
Comment (roldanbob) [inline documentation/modules/ROOT/pages/connectors/db2.adoc:3049]:
```suggestion
|Specifies the time, in milliseconds, that the connector waits for a query to complete.
Set the value to `0` (zero) to remove the timeout limit.
```

File: documentation/modules/ROOT/pages/connectors/informix.adoc [hunk 2]
```diff
@@ -2648,6 +2648,10 @@ You can define tags to identify particular application instances, environments,
 |`-1`
 |The maximum number of retries on retriable errors (e.g. connection errors) before failing (-1 = no limit, 0 = disabled, > 0 = num of retries).
 
+|[[informix-property-database-query-timeout-ms]]<<informix-property-database-query-timeout-ms, `database.query.timeout.ms`>>
+|`600000`
+|Time to wait for a query to execute, given in milliseconds. Defaults to 600 seconds (600,000 ms); zero means there is no limit.
```
Comment (roldanbob) [inline documentation/modules/ROOT/pages/connectors/informix.adoc:2653]:
```suggestion
|`600000` (10 minutes)
|Specifies the time, in milliseconds, that the connector waits for a query to complete.
Set the value to `0` (zero) to remove the timeout limit.
```

File: documentation/modules/ROOT/pages/connectors/mysql.adoc [hunk 3]
```diff
@@ -3920,6 +3920,10 @@ Set one of the following options:
 `fail`::  An exception reports that conversion failed because the data type of the field did not match the schema type, and indicates that it might be necessary to restart the connector in `schema _only_recovery` mode to enable a successful conversion.
 `warn`:: The connector writes a `null` value to the event field for the column that failed conversion, writes a message to the warning log . +
 `skip`::  The connector writes a `null` value to the event field for the column that failed conversion, and writes a message to the debug log.
+
+|[[mysql-property-database-query-timeout-ms]]<<mysql-property-database-query-timeout-ms, `database.query.timeout.ms`>>
+|`600000`
+|Time to wait for a query to execute, given in milliseconds. Defaults to 600 seconds (600,000 ms); zero means there is no limit.
```
Comment (roldanbob) [inline documentation/modules/ROOT/pages/connectors/mysql.adoc:3926]:
```suggestion
|`600000` (10 minutes)
|Specifies the time, in milliseconds, that the connector waits for a query to complete.
Set the value to `0` (zero) to remove the timeout limit.
```

File: documentation/modules/ROOT/pages/connectors/postgresql.adoc [hunk 4]
```diff
@@ -3696,6 +3696,9 @@ User intervention is required to restart the connector.
 `> 0`:: The connector restarts automatically until it reaches the specified maximum number of retries.
 After the next failure, the connector stops, and user intervention is required to restart it.
 
+|[[postgresql-property-database-query-timeout-ms]]<<postgresql-property-database-query-timeout-ms, `database.query.timeout.ms`>>
+|`600000`
+|Time to wait for a query to execute, given in milliseconds. Defaults to 600 seconds (600,000 ms); zero means there is no limit.
```
Comment (roldanbob) [inline documentation/modules/ROOT/pages/connectors/postgresql.adoc:3701]:
```suggestion
|`600000` (10 minutes)
|Specifies the time, in milliseconds, that the connector waits for a query to complete.
Set the value to `0` (zero) to remove the timeout limit.
```

File: documentation/modules/ROOT/pages/connectors/sqlserver.adoc [hunk 5]
```diff
@@ -3269,6 +3269,9 @@ After the next failure, the connector stops, and user intervention is required t
 * `function`: The data is queried by calling `cdc.[fn_cdc_get_all_changes_#]` function. This is the default mode.
 * `direct`: Makes the connector to query change tables directly. Switching to `direct` mode and creating an index on `(\\__$start_lsn ASC, __$seqval ASC, __$operation ASC)` columns for each change table significantly speeds up querying CDC data.
 
+|[[sqlserver-property-database-query-timeout-ms]]<<sqlserver-property-database-query-timeout-ms, `database.query.timeout.ms`>>
+|`600000`
+|Time to wait for a query to execute, given in milliseconds. Defaults to 600 seconds (600,000 ms); zero means there is no limit.
```
Comment (roldanbob) [inline documentation/modules/ROOT/pages/connectors/sqlserver.adoc:3274]:
```suggestion
|`600000` (10 minutes)
|Specifies the time, in milliseconds, that the connector waits for a query to complete.
Set the value to `0` (zero) to remove the timeout limit.
```

**Comment by mfvitale** [comment]
@roldanbob thanks for your suggestions, do you think we are good now?

**Review by roldanbob — APPROVED** [review]
LGTM, @mfvitale! Thanks.

**Pull request merged**